### ISSUE TITLE
feature: configure display of language names in languange switcher 

### ISSUE THREAD

**Issue by mayuxi** [gohugo-ananke/ananke#707]
Hi!

A little suggestion. Now Ananke uses `{{ .Lang }}` as language link in `layouts/partials/i18nlist.html`, which makes language switcher links like that:

`en pt-BR he ru`

I suggest using `.Site.Language.LanguageName` instead, so the user can customize the language names in his config file. For example, it could be full names (English, Português, Русский), or even flag emojis (🇬🇧 🇧🇷 🇷🇺).

Sorry, I'm new to Hugo and Ananke, here is my suggested code for **layouts/partials/i18nlist.html**:

before

`<a class="hover-white no-underline white-90" href="{{ .RelPermalink }}">{{ .Lang }}</a>`

after

`<a class="hover-white no-underline white-90" href="{{ .RelPermalink }}">{{ .Site.Language.LanguageName | default .Lang }}</a>`

**Comment by davidsneighbour** [comment]
I agree; this should be a configuration option. For the time being, you can override the theme file by adding a copy to your own repository under `layouts/partials/i18nlist.html` and adding your changes there.
